CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 	## __WORK IN PROGRESS__
 	(at the beginning of a new line)
 -->
+
+## __WORK IN PROGRESS__
+* (hacki11) Support git-ssh in devcontainer (#1136)
```

Comment:
There should be a link to your migration guide, see for example the `2.6.0` changelog.

Reply:
done